python/mxnet/gluon/utils.py

```diff
@@ -430,3 +432,44 @@ def shape_is_known(shape):
         assert dim_size > unknown_dim_size, "shape dimension size cannot be less than {}, while " \
                                             "received {}".format(unknown_dim_size, dim_size)
     return True
+
+
+def _check_same_symbol_type(symbols):
+    """Check whether all the symbols in the list are of the same type.
+    Raise type error if the types are different. Return the class of
+    the symbols."""
+    from ..symbol.numpy import _Symbol as np_symbol
+    from ..symbol import Symbol as nd_symbol
+    is_np_sym = bool(isinstance(symbols[0], np_symbol))
```

Comment:
Looks like `bool` is unnecessary.

Reply:
If I remember it correctly, seems it's added intentionaly for passing CI failure caused by different python versions, but my memory may be wrong...

Reply:
Anyway, should be okay to keep the bool

Reply:
`bool` kept for now.

Reply:
I just checked the commit history. It was added to fix another pylint error, but the `bool` should be safe to be removed.

Reply:
Removed locally, will commit later.